modules/access-management/variables.tf

```diff
@@ -0,0 +1,1535 @@
+    })))
+    resource_attributes = optional(list(object({
+      name     = string
+      value    = string
+      operator = optional(string)
+    })))
+  }))
+  description = "A map of policies for the Administrator Network access group, has a set of default policies that can be overridden"
+  default = {
+    "endpoint-gateway" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.endpoint-gateway"
+      }]
+      tags = []
+    }
+    "floating-ip" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.floating-ip"
+      }]
+      tags = []
+    }
+    "load-balancer" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.load-balancer"
+      }]
+      tags = []
+    }
+    "network-acl" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.network-acl"
+      }]
+      tags = []
+    }
+    "public-gateway" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.public-gateway"
+      }]
+      tags = []
+    }
+    "subnet" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.subnet"
+      }]
+      tags = []
+    }
+    "virtual-network-interface" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.virtual-network-interface"
+      }]
+      tags = []
+    }
+    "vpn" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.vpn"
+      }]
+      tags = []
+    }
+    "vpn-server" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.vpn-server"
+      }]
+      tags = []
+    }
+    "vpc" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "is.vpc"
+      }]
+      tags = []
+    }
+    "context-based-restrictions" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "context-based-restrictions"
+      }]
+      tags = []
+    }
+    "context-based-restrictions.zone" = {
+      roles = ["Administrator"]
+      resources = [{
+        service = "context-based-restrictions.zone"
+      }]
+      tags = []
+    }
+  }
+}
+
+variable "admin_network_ag_dynamic_rules" {
+  type = map(object({
+    expiration        = number
+    identity_provider = string
+    conditions = list(object({
+      claim    = string
+      operator = string
+      value    = string
+    }))
+  }))
+  description = "A map of dynamic rules for the admin network access group"
+  default     = {}
+}
+
+variable "admin_network_ag_add_members" {
+  type        = bool
+  description = "Enable this to add members to the admin network group"
+  default     = false
+}
+
+variable "admin_network_ag_description" {
+  type        = string
+  description = "Description of the admin network access group"
+  default     = null
+}
+
+variable "admin_network_ag_tags" {
+  type        = list(string)
+  description = "Tags that should be applied to the service"
```

Comment:
`The list of tags that you want to associated with your admin network access group`